idd/Energy+.idd.in

```diff
@@ -67789,6 +68421,8 @@ ThermalStorage:Ice:Simple,
       \type alpha
       \required-field
       \reference IceThermalStorageEquipment
+      \reference-class-name validBranchEquipmentTypes
```

Comment:
@Nigusse I'm surprised that ice storage objects are not also included in validPlantEquipmentTypes.  I assume you confirmed that in source code?